string.go

```diff
@@ -101,3 +101,18 @@ func TruncateString(str string, num int) string {
 	}
 	return bnoden
 }
+
+func EscapeQuote(in string) string {
+	res := []byte{}
+	const (
+		doubleQuoteASCII = 34
+		backSlashASCII   = 92
+	)
+	for i := 0; i < len(in); i++ {
+		if in[i] == doubleQuoteASCII && ((i > 0 && in[i-1] != backSlashASCII) || i == 0) {
+			res = append(res, backSlashASCII)
+		}
+		res = append(res, in[i])
+	}
+	return string(res)
+}
```

Comment:
Kalau pake `index()` kira2 gimana jung? Jadi idenya:
1. Find the index of the quote
2. Check if the previous char is backslash
3. If not, add the backslash
4. Otherwise ignore
https://appdividend.com/2020/04/17/golang-string-index-how-to-find-index-of-substring-in-go/

Reply:
nvm udah di bahas sama mas tedja, ini gak jadi ya jung